### ISSUE TITLE
cannot find function `fstype_imp` in this scope

### ISSUE THREAD

**Issue by yonas** [facebook/sapling#154]
```
$ make oss
...
   Compiling iovec v0.1.4
   Compiling time v0.1.44
   Compiling fsinfo v0.1.0 (/data/git/sapling/eden/scm/lib/fsinfo)
error[E0425]: cannot find function `fstype_imp` in this scope
   --> lib/fsinfo/src/lib.rs:390:5
    |
390 |     fstype_imp(path).with_context(|| format!("Cannot determine filesystem type for {:?}", path))
    |     ^^^^^^^^^^ not found in this scope

   Compiling num_threads v0.1.6
   Compiling mio v0.7.14
warning: unused import: `anyhow::Context`
  --> lib/fsinfo/src/lib.rs:12:5
   |
12 | use anyhow::Context;
   |     ^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0425`.
warning: `fsinfo` (lib) generated 1 warning
error: could not compile `fsinfo` due to previous error; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
error: compilation of Rust target 'hgmain' failed
make: *** [Makefile:75: oss] Error 1
```

OS: `FreeBSD 13.1-RELEASE-p3 amd64`

**Comment by quark-zju** [comment]
What operating system are you using?

**Comment by quark-zju** [comment]
@yonas Right now I don't have a FreeBSD system at hand to test it out. This seems to be not too difficult to implement properly - might be similar to macOS. Feel free to send a PR implementing it!

**Comment by yonas** [comment]
Fixed in #514

**Comment by yonas** [comment]
Fixed in #514

**Comment by yonas** [comment]
Fixed in #514

**Comment by yonas** [comment]
Fixed in #514
